src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -74,8 +76,16 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeFramework Include="GSS" />
     </ItemGroup>
 
+    <Message Text="IlcPath = $(IlcPath)" />
```

Comment:
You can get this path from the verbose log. Printing it all the time adds unnecessary clutter to the default output.

Reply:
Ok, will remove the prints.